ostis-web-platform.ini

```diff
@@ -8,9 +8,9 @@ update_period = 1800
 repo_path = kb.bin
 extensions_path = sc-machine/bin/extensions
 
-log_type = File
+log_type = Console
 log_file = sc-machine/sc-memory.log
-log_level = Info
+log_level = Debug
```

Comment:
Do not change log type and level. Change SC_LOG_DEBUG to SC_LOG_INFO in SC-component-manager commands